tests/databases/test_redshift.py

```diff
@@ -189,10 +189,12 @@ def test_load_file_from_cloud_to_table(database_table_fixture):
     database, target_table = database_table_fixture
     database.load_file_to_table(
         File(
-            "s3://astro-sdk/data_redshift/", conn_id="aws_conn", filetype=FileType.CSV
+            "s3://astro-sdk-redshift/data_redshift/",
+            conn_id="aws_conn",
+            filetype=FileType.CSV,
         ),
         target_table,
-        {},
+        use_native_support=False,
```

Comment:
@pankajkoti why is this - `use_native_support=False` here? what are we testing?

Reply:
This is an existing test which was needed for testing load from a S3 bucket containing files and sub_folders; where the with the default option dataframe infers the schema and loads data into corresponding table.